### PR TITLE
Add a pull request template

### PR DESCRIPTION
**Pull request by igordcsouza**
## Description
Adicionar um template de PR para facilitar quem não sabe o que deveria escrever.

## Motivation and Context
Facilitar a vida de pessoas que não tem pratica em fazer PR.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by athityakumar** [comment]
I feel that something like this would do a better job...

```markdown
### Description

A brief description of what the PR does. Do include the relevant issue(s) and PR(s) (say, if this PR is a continuation of another PR).

### Screenshots

Some screenshot(s) to support your changes.

### Type of change

- [ ] New feature
- [ ] Bug fix for existing feature
- [ ] Code quality improvement
- [ ] Addition or Improvement of tests
- [ ] Addition or Improvement of documentation
```

... which renders like the below.

### Description

A brief description of what the PR does. Do mention the relevant issue(s) and PR(s) (say, if this PR is a continuation of another PR). Adding a sample use-case would be better.

### Screenshots

Some screenshot(s) to support your changes.

### Type of change

- [ ] New feature
- [ ] Bug fix for existing feature
- [ ] Code quality improvement
- [ ] Addition or Improvement of tests
- [ ] Addition or Improvement of documentation

**Comment by athityakumar** [comment]
Ping @igordcsouza

**Comment by igordcsouza** [comment]
pong...  :) 
I'm think... maybe the "Some screenshot(s) to support your changes." should be a comment lines?

**Comment by athityakumar** [comment]
@igordcsouza - Yep, comment that line and we're ready to merge. 😄

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+### Description
+
+A brief description of what the PR does. Do include the relevant issue(s) and PR(s) (say, if this PR is a continuation of another PR).
```
Comment (athityakumar) [inline PULL_REQUEST_TEMPLATE.md]:
```markdown
<!-- A brief description of what the PR does. Tag the relevant Issues and PRs below. -->

- Relevant Issues : 
- Relevant PRs (if any) :
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Screenshots
+
+Some screenshot(s) to support your changes.
```
Comment (athityakumar) [inline PULL_REQUEST_TEMPLATE.md]:
```markdown
<!-- Some screenshot(s) to support your changes. -->
```

**Pull request merged**